integration/test_redis_specific.py

```diff
@@ -45,13 +54,22 @@ def test_redis_driver_write(redis):
 
     assert data_strings == {"col1": '0'}
 
+    for key in driver.redis.scan_iter(f'*storey:{table_name}*'):
+        driver.redis.delete(key)
```

Comment:
If the point is cleanup, better put this in teardown or in a finally clause.

Reply:
ok